### PR TITLE
Internship experience blog

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Description**

This PR : Internship experience blog with layer5

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Internship Experience with Layer5 - Sudhanshu Dasgupta"
+subtitle: "Unleashing the Journey: From Open-Source Enthusiast to Internship Success"
+date: 2023-06-15 10:30:05 -0530
+author: Sudhanshu Dasgupta
+category: "Community"
+description: "This is a blog sharing my internship and open-source experience with Layer5"
+category: Community
```
Comment (vishalvivekm) [inline src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:8]:
```suggestion

```

Comment (vishalvivekm) [reply src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:8]: build is failing please remove the suggested line of code @sudhanshutech

File: src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Internship Experience with Layer5 - Sudhanshu Dasgupta"
+subtitle: "Unleashing the Journey: From Open-Source Enthusiast to Internship Success"
+date: 2023-06-15 10:30:05 -0530
+author: Sudhanshu Dasgupta
+category: "Community"
```
Comment (vishalvivekm) [inline src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:6]:
```suggestion

```
remove this line of code, build is failing due to duplicacy at 6th and 8th line

File: src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+description: "This is a blog sharing my internship and open-source experience with Layer5"
```
Comment (Chadha93) [inline src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:7]: No need to mention - "This is a blog" @sudhanshutech

File: src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+Venturing forth, I attended my very first Newcomers meet, a gateway to understanding the vibrant community and its wide array of projects. It was an enriching experience, immersing myself in the ethos of collaboration and knowledge sharing that defined the community's spirit.
+Taking a small yet significant step forward, I eagerly embraced a task assigned to me.
+I understood that even the tiniest actions can pave the way for monumental achievements. But my journey didn't halt there; fueled by my unwavering commitment, continuous skill development, and active engagement with the community, I witnessed remarkable progress.
+And then, within a mere two months, my dedication and contributions were acknowledged, leading to an incredible milestone—I was awarded an internship at Layer5.
+</p>
+
+<h2 id="experience">
+ My Experience of Internship with Layer5🚶
+</h2>
+
```
Comment (Chadha93) [inline src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:44]: @sudhanshutech Let's add some images if you prefer.

Comment (sudhanshutech) [reply src/collections/blog/2023/2023-06-15-internship-experience-blog-sudhanshu-dasgupta/index.mdx:44]: sure @Chadha93

**Comment by sudhanshutech** [comment]
does we need images here @Chadha93

**Comment by Chadha93** [comment]
> does we need images here @Chadha93

You can toss in a graphic if you like.

**Comment by Chadha93** [comment]
Build is failing.... @sudhanshutech

**Comment by leecalcote** [comment]
@sudhanshutech, @Chadha93, publish when ready.

**Comment by Chadha93** [comment]
@sudhanshutech Build failing....

**Comment by l5io** [comment]
🚀 Preview for commit e905b4b07044563c5f446d78a1d98f70fd7ee5cb at: https://64c4cc3da105034ea1091b14--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f0f9b46812a34f3248df3657d0ede37d72c65584 at: https://64c7f22de9c47e2eba93ac1c--layer5.netlify.app

**Pull request merged**